code/datums/brain_damage/psychological.dm

```diff
@@ -0,0 +1,330 @@
+/*TODO:
+	PTSD
```

Comment:
Edge is about to make the old folks home alot more scarier

Reply:
i forgot to delet that comment kek